docs/tutorials/projects/modernize-retro-robot.md

```diff
@@ -3,10 +3,608 @@ title: "Modernize a 1980s robot"
+
+Navigate to your robot's [Control](/manage/fleet/robots/#control) tab.
+Click on the base panel and use the arrows to control the base.
+Ensure the motors on both sides are working, and that the wheels are spinning as expected.
+If the wheels on a given side are spinning in the opposite direction of what you would expect, go back to the **Config** tab, toggle `Direction Flip` for the appropriate motor, **Save config** and repeat the test steps.
+
+{{<video webm_src="../../img/maiv/maiv_base_test.webm" mp4_src="../../img/maiv/maiv_base_test.mp4" alt="MAIV base test movements" max-width="300px">}}
+
+From the **Control** tab you can also test the individual motors directly.
+First select the desired direction and power percent, start the motor by clicking `RUN` and stop the motor by clicking `STOP`.
+
+### MAIV's head
+
+In order to give MAIV a bit more personality and capability, let's wire MAIV's neck motor and eyes.
+We'll also add a camera so MAIV can see the world.
+
+#### Convert MAIV's eyes to LEDs
+
+Because MAIV's torso is already open, the neck and head is detached.
+Turn the head over, locate and unscrew the two screws holding MAIV's face plate.
+You will be replacing the current incandescent bulbs with more energy-efficient LEDs that can be controlled directly from your Raspberry Pi.
+In order to access the eye wiring, locate the screws on the underside of the head and unscrew them.
+
+<div class="td-max-width-on-larger-screens">
+<img src="../../img/maiv/maiv_eyes.jpg" alt="Omnibot MAIV eye plate." class="alignleft" title="Omnibot MAIV eye plate." style="max-width: 250px;" />
+</div>
+
+Remove the eye shield, unscrew the incandescent bulbs and screw in the LED bulbs.
+
+The original wires to the eye bulb sockets won't be long enough to reach the Raspberry Pi.
+Cut the original wires (leave some length) and strip the ends.
+You will control both eyes at once, so use a 3-terminal quick connect to connect both eyes positive wires and a third long breadboard wire.
+Repeat the same with another 3-terminal quick connect for the negative wires.
+
+Now, run the long breadboard wires to through the robot neck, torso, and into the base.
+
+With the robot powered off:
+
+* Plug the positive breadboard wire into pin 1 on the Raspberry Pi (3.3V).
+* Plug the negative breadboard wire into pin 18 (GPIO 24) on the Pi.
+
+Power your robot back on to test the eyes.
+You will use GPIO directly to control the eyes through the `board` component you already configured.
+Once viam-server is running (it will take a minute or so to initialize), go to the **Control** tab and open the *local* board card.
+From here, you can test MAIV's eyes by setting GPIO pin 18 to high (to turn the eyes on) or low (to turn the eyes off).
+
+{{<video webm_src="../../img/maiv/maiv_eye_test.webm" mp4_src="../../img/maiv/maiv_eye_test.mp4" alt="MAIV eye flashing test" max-width="300px">}}
```

Comment:
just curious: is it the motors that are loud?

Reply:
Good question!  I was trying to remember that myself (I recorded these videos a month ago) - I think it was perhaps some fan or heater I had running in the background.  The motors are loud in the Omnibot but they were not on here....